spec/features/admin/sort_view_in_organisation_list_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe 'sorting the columns in the organisation list' do
+  context 'when admin views the names of organisations' do
+    let!(:organisation_1) { create(:organisation, name: "ABC", created_at: Date.yesterday) }
+    let!(:organisation_2) { create(:organisation, name: "DEF", created_at: 1.month.ago) }
+    let!(:organisation_3) { create(:organisation, name: "XYZ", created_at: 2.years.from_now) }
+
+    let!(:user) { create(:user, :confirmed, admin: true) }
+
+    before do
+      sign_in_user user
+      visit root_path
+    end
+
+    it 'can sort the organisations by names in descending order' do
+      click_link 'Name'
+
+      expect(page.body).to match(/XYZ.*DEF.*ABC/m)
+    end
+
+    context 'can toggle the order of the dates' do
```

Comment:
I would describe behaviour by describing what the component *does*, rather than what it "should" or "can" do, as you end up with less noise around the key information.

I'm a little confused by which parts of this are `context` and which are `it` blocks, I don't think the doc output would be as clear as it could be.

Think the structure should be something like:
```ruby
context 'showing the list of organisations'
  context 'by default'
    it 'is sorted by name, ascending'
 
  context 'with name selected once'
    it 'is sorted by name, descending'

  context 'with date selected once'
    it 'is sorted by date, from oldest to newest'

  context 'with date selected twice'
    it 'is sorted by date, from newest to oldest'
```